### PR TITLE
Adding Contributing.md

### PR DESCRIPTION
**Pull request by gwincr11**
Motivation:
  - Started writing a contributing guide

Related Issues:
  - #36

### CODE REVIEW

**Comment by gwincr11** [comment]
Just a thought but it maybe nice to have issue templates for bug reports and feature requests we could link to...

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Contributing
+Welcome! We are thrilled you are interested in contributing to Covasim. This document will help you get started.
+
+## Notices
+Contributions to this project are [released](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) to the public under the [project's open source license](LICENSE).
+
+
+Please note that this project is released with a [Contributor Code of Conduct](https://github.com/InstituteforDiseaseModeling/covasim/blob/master/CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
```
Comment (jules2689) [inline CONTRIBUTING.md:8]: Why is this linked to master, but license is not?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	- [Covasim Readme](https://github.com/InstituteforDiseaseModeling/covasim/blob/master/covasim/README.md) Simulation parameters.
+	- [Web App Readme](https://github.com/InstituteforDiseaseModeling/covasim/blob/master/covasim/webapp/README.md) Running the web application.
+	- [Docker Readme](https://github.com/InstituteforDiseaseModeling/covasim/blob/master/docker/README.md) Information regarding the docker setup.
+	- [Tests Readme](https://github.com/InstituteforDiseaseModeling/covasim/tree/master/tests) Running tests locally.
+- [Changelog](https://github.com/InstituteforDiseaseModeling/covasim/blob/master/CHANGELOG.md) 
+
+## Contribution Types
+This is a fast moving project with many opportunities to contribute across the project. We welcome the following types of contributions:
+
+1. Issues:
```
Comment (jules2689) [inline CONTRIBUTING.md:26]: Should we add issue and or templates?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+* **perform a cursory search** to see if there's [an existing issue](https://github.com/InstituteforDiseaseModeling/covasim/issues) covering your feedback. If there is one and the issue is still open, **add a :+1: reaction** on the issue to express interest in the issue being resolved. That will help the team gauge interest without the noise of comments which trigger notifications to all watchers. Comments should be used only if you have new and useful information to share.
+
+When opening an issue for a feature request:
+
+* **use a clear and descriptive title** for the issue to identify the problem.
+* **include as many details as possible in the body**. Explain your use-case, the problems you're hitting and the solutions you'd like to see to address those problems.
+
+When opening an issue for a bug report, explain the problem and include additional details to help maintainers reproduce the problem:
+
+* **describe the exact steps which reproduce the problem** in as many details as possible. When listing steps, don't just say what you did, but explain how you did it.
```
Comment (jules2689) [inline CONTRIBUTING.md:50]: Issue template could have this all, which would dramatically shorten this document

**Comment by cliffckerr** [comment]
@jules2689 @gwincr11 Just added issue templates. Thank you for the suggestion, these are awesome!

File: CONTRIBUTING.md (hunk 3 above)
Comment (cliffckerr) [reply CONTRIBUTING.md:50]: even with issue templates, i don't think it hurts to have duplication -- i'd leave as is

File: CONTRIBUTING.md (hunk 2 above)
Comment (cliffckerr) [reply CONTRIBUTING.md:26]: YES :)

File: CONTRIBUTING.md (hunk 3 above)
Comment (gwincr11) [reply CONTRIBUTING.md:50]: Yes, I mentioned that in chat and the related issue.

**Pull request merged**